resources/yki/queries.sql

```diff
@@ -665,12 +665,22 @@ WHERE re.id = :id
   AND re.state = 'SUBMITTED'
   AND p.external_user_id = :external_user_id;
 
--- name: update-started-registrations-to-expired<!
+-- name: select-started-registrations-to-expire
+SELECT id FROM registration
+WHERE state = 'STARTED' AND (started_at + interval '30 minutes') < current_timestamp;
```

Comment:
scheduled_tasks `handle-started-registration-expired-test` testin alle vois lisätä myös tsekkauksen, että aloitettuja rekisteröintejä ei merkata vielä expired tilaan jos alle 30min kulunut. Tällä hetkellä siellä näyttäis tarkistettavan vain että tunti sitten luotu rekisteröinti expiroidaan.

Reply:
Lisäsin uuden, erillisen testin.